### ISSUE TITLE
Comparison to ConfidNet

### ISSUE THREAD

**Issue by ssakhavi** [daintlab/confidence-aware-learning#10]
Hi, Great Paper!

Can I ask why you haven't compared your results with ConfidNet(Addressing Failure Prediction by Learning Model Confidence)?

Also, do you think there is a version of your algorithm which can be used to estimate confidence **post training**? Considering the fact that many models are available out there but are not confidence-aware.

**Comment by JihyoKim00** [comment]
Hello, Thank you for your interest!

We compared our method with ConfidNet but there was no enough time gap between the conference deadline and the time when ConfidNet published, so the comparison results could not be put in the paper. CRL shows better performance than ConfidNet, and it is reasonable results because in large network in DenseNet, the true class probabilities of training data will be almost 1. It makes the learning trivial. 

One of the our method's advantages is that making well-trained classifier which quantify predictive probability better without two-step training. Two-step training is also a good strategy, but we observed that it needs more training epochs and leads loss of efficiency. You can do post training with CRL. It is not as good as the one-step training, but it also can give performance improvement than baseline. It would be good to compare the post training performance with various methods.

Please let me know if I misunderstood your questions. Thank you!

**Comment by ssakhavi** [comment]
Sorry for the late reply.


Thanks for clarifying.
